internal/commands/reminder.go

```diff
@@ -23,19 +23,33 @@ func requestStatus(ctx context.Context, client bot.Client, logger log.Logger, re
 		if err != nil {
 			logger.Error(
 				ctx,
-				"command/reminder.requestStatus GetIncident error",
-				log.NewValue("channelID", incident.ChannelId),
+				log.Trace(),
+				log.NewValue("reason", "GetIncident"),
+				log.NewValue("channelId", incident.ChannelId),
+				log.NewValue("channelName", incident.ChannelName),
 				log.NewValue("error", err),
 			)
 			return
 		}
 
+		logger.Info(
+			ctx,
+			log.Trace(),
+			log.NewValue("action", "running"),
+			log.NewValue("channelID", incident.ChannelId),
+			log.NewValue("channelName", incident.ChannelName),
+		)
+
```

Comment:
Should this be log here? In this part of the code, we are not running the reminder yet.

Reply:
It is running the func requestStatus

Reply:
Agreed!